amr-wind/wind_energy/actuator/ActSrcLineOp.H

```diff
@@ -116,7 +116,17 @@ void ActSrcOp<ActTrait, ActSrcLine>::operator()(
             const auto pos_ip = wt * pos[ip] + (1.0 - wt) * opos[ip];
             const auto dist = cc - pos_ip;
             const auto dist_local = tmat[ip] & dist;
-            const auto gauss_fac = utils::gaussian3d(dist_local, eps[ip]);
```

Comment:
@mbkuhn and @tonyinme I am not sure if the 'dist_local' and 'eps' vectors are consistent. When using the x-z plane for a 2D airfoil (spanwise direction is the y direction) and different epsilon values for the spanwise&thickness directions the thickness value appeared in the y direction rather than the z direction (and the spanwise value in the z direction rather than the y direction).
For the 2D Gaussian I flipped the order now, but we should probably check also the 3D Gaussian. Maybe I am wrong and I miss some detail here.

Reply:
I see, yes, that makes sense. 
When I tested this a few years ago, I remember the chord direction being aligned correctly, but I did not pay attention to the other 2 directions. So it makes sense that they are inverted but I missed it.

Reply:
@ET14 this was a good catch. For the sake of the vector and tensor operations in the code, the local coordinates are (chord, span, thickness), which is the way dist_local and eps are laid out when used in ActSrcLineOp, but the convention for specifying epsilon is (chord, thickness, span). Therefore, the thickness and span components need to switch places after being input and before being used. The problem was that for the fixed wing, this switch was not taking place for eps_inp, which explains the mismatch you were seeing. Corrected now.

Reply:
@mbkuhn Another thought related to this. I think the transformation tmat is only calculated here (https://github.com/Exawind/amr-wind/blob/02feaf5699dcc75a124a6361178b1217c49f8d76/amr-wind/wind_energy/actuator/wing/wing_ops.cpp#L43) during the init phase with the initial pitch angle. However, now the pitch angle is time dependent. Should we update tmat at each time step?

Reply:
Ah yes, great catch. By the looks of it, the pitch is only used at initialization for this. This should be clear if you set epsilon to be different in chordwise and thickness directions and pitch the wing -- the Gaussian won't pitch with it, even though the code should still be calculating the angle of attack correctly. the grid.orientation variable needs to be updated when the pitch is updated in the ComputeForceOp operator within wing_ops.H.

Reply:
almost forgot to go back and fix this. the transformation matrix now updates every compute force step, and the tilting of the force distribution is obvious when an anisotropic epsilon distribution is specified.